platform.lua

```diff
@@ -20,11 +20,11 @@ this.platform = (
                 or h.is_mac() and this.Platform.macos
                 or this.Platform.gnu_linux
 )
-this.default_video_folder = utils.join_path(
+this.default_video_folder = io.popen("xdg-user-dir VIDEOS"):read("*l") or utils.join_path(
```

Comment:
Why not use subprocess here? See `mp.command_native`.

Reply:
Okay, done. like this?

Reply:
>Is using mpv's own execute better than lua's os.execute or io.popen?

with subprocess you don't have to worry about arguments being split incorrectly, e.g. when an argument contains spaces.

Reply:
ahhh i see